app/src/features/Batches/BatchCreate.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+
+import {
+  Button,
+  Chip,
+  FormControl,
+  Input,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PlayIcon from "@material-ui/icons/PlayCircleOutline";
+import { useTranslation } from "react-i18next";
+import { useParams } from "react-router-dom";
+
+import Alert from "common/components/Alert";
+import {
+  useApiResourcesListQuery,
+  useApiBatchesCreateMutation,
+} from "services/api/endpoints";
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
```

Comment:
You should use the property theme.spacing(1) which is the same size as ITEM_PADDING_TOP